File: wallendar-app/app/src/main/java/com/td/wallendar/group/ui/GroupView.java
package com.td.wallendar.group.ui;

import com.td.wallendar.models.Group;
import com.td.wallendar.models.GroupHistory;

import java.util.List;

public interface GroupView {

    void bindGroup(Group group);

    void bindGroupHistory(List<GroupHistory> historic);

}

File: wallendar-app/app/src/main/java/com/td/wallendar/group/ui/GroupView.kt
package com.td.wallendar.group.ui

import com.td.wallendar.models.Group
import com.td.wallendar.models.GroupHistory

interface GroupView {
    open fun bindGroup(group: Group)
    open fun bindGroupHistory(historic: MutableList<GroupHistory>)
    open fun onGroupLoadError()
}